_posts/Installation/0000-01-02-arm_install.md

```diff
@@ -31,21 +32,20 @@ Clone the repo: `git clone git@github.com:LibrePhotos/librephotos-docker.git`
 
 `cp librephotos.env .env`
 
-Open `.env` in your favorite text editor and make the required changes. 
+Open `.env` in your favorite text editor and make the required changes.
 
 Do not forget to create the directories you specified in the `.env` file if they do not exist.
 
 Run `docker-compose -f docker-compose.arm.yml up -d`
 
 **If you do not follow the above command and use just `docker-compose up -d` you are going to have a bad time**
 
-
 You should have librephotos accessible after a few minutes of bootup on: [localhost:3000](http://localhost:3000) unless you changed it in the .env file.
 User is admin, password is admin unless you changed it in the .env file. It is recommended you change the admin username and password if Libre Photos is going to be publicly accessible via the ``.env`` file.
 
 ## First steps after setting up
 
-You need to log in as the admin user, and set up the directory for the users. To do this, click the top right button, and go to "Admin Area". On this page, it will show a list of users, and manually set the "Scan Directory" for the desired user. Only an admin can do this. And then you can go to Dashboard - Library and click the Green "Scan photos (file system)" button. If you have a Nextcloud instance, you can also input this in the Dashboard-Library page. Once logged in (the little circle next to "Nextcloud Scan Directory will be green), you can choose a top level directory in your logged in Nextcloud account. Once this works, you can click the blue "Scan photos (Nextcloud)". The backend system will copy the contents of the Nextcloud directory you specified. 
+You need to log in as the admin user, and set up the directory for the users. To do this, click the top right button, and go to "Admin Area". On this page, it will show a list of users, and manually set the "Scan Directory" for the desired user. Only an admin can do this. And then you can go to Dashboard - Library and click the Green "Scan photos (file system)" button. If you have a Nextcloud instance, you can also input this in the Dashboard-Library page. Once logged in (the little circle next to "Nextcloud Scan Directory will be green), you can choose a top level directory in your logged in Nextcloud account. Once this works, you can click the blue "Scan photos (Nextcloud)". The backend system will copy the contents of the Nextcloud directory you specified.
```

Comment:
```suggestion
You need to log in as the admin user, and set up the directory for the users. To do this, click the top right button, and go to `Admin Area`. On this page, it will show a list of users. Manually set the `Scan Directory` for the desired user. Only an admin can do this. Next, go to `Dashboard`, and then `Library`; and click the green `Scan photos (file system)` button. If you have a Nextcloud instance, you can also input this in the `Dashboard`>`Library` page. Once logged in (the little circle next to `Nextcloud Scan Directory` will be green), you can choose a top level directory in your logged in Nextcloud account. Once this has been configured, you can click the blue `Scan photos (Nextcloud)` button. The backend system will copy the contents of the specified Nextcloud directory to the local filesystem.
```
Formatting and readability